autowarefoundation/autoware
# Add SICK LMS511 driver

## Status
DEVELOPMENT

## Description
Added SICK LMS511 driver.

## Todos

## Steps to Test or Reproduce
1. Open "config" next to the SICK LMS511 in Sensing tab.
1. Input the IP address of the SICK LMS511
1. Launch the node
1. Open RViz and check if the scan displays (Frame ID: sick, topic: /scan)
